assets/scss/components/elements/blog/_single.scss

```diff
@@ -125,3 +131,172 @@
 	color: #fff !important;
   }
 }
+
+.nv-post-cover {
+  min-height: var(--height);
+  color: var(--color, var(--nv-text-dark-bg));
+  padding: var(--padding);
+  display: flex;
+  position: relative;
+
+  &.has-image{
+	background-size: cover;
+	background-repeat: no-repeat;
+	background-position: center;
+  }
+
+  .nv-title-meta-wrap {
+	display: inline-flex;
+	flex-direction: column;
+	flex-wrap: wrap;
+	z-index: 1;
+	&.reversed{
+	  flex-direction: column-reverse;
+	}
+	&.is-boxed{
+	  padding: var(--boxedTitlePadding);
+	  background-color: var(--boxedTitleBackground, var(--nv-light-bg));
+	}
+  }
+
+  .nv-meta-list{
+	li, a {
+	  color: var(--color, var(--nv-text-dark-bg) );
+	}
+  }
+
+  .container{
+	z-index: 1;
+	display: flex;
+  }
+
+}
+
+.nv-overlay {
+  background-color: var(--background,var(--nv-dark-bg));
+  mix-blend-mode: var(--blendMode,normal);
+  opacity: var(--opacity);
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+}
+
+.nv-comments-wrap.is-boxed {
+  padding: var(--boxedCommentsPadding);
```

Comment:
Would this conflict with anything if we used `--padding`?  Same for the variables below:

`--boxedCommentsBackground` => `--bgColor`
`--boxedCommentsColor` => `--color`